### PR TITLE
PageIndex Generation Reverted

### PR DESCRIPTION
**Pull request by fcuneyt**

### CODE REVIEW

File: src/SimpleMvcSitemap.Sample/SampleBusiness/ProductSitemapConfiguration.cs [hunk 1]
```diff
@@ -7,15 +7,18 @@ public class ProductSitemapConfiguration : ISitemapConfiguration<Product>
...
         {
             _urlHelper = urlHelper;
             Size = 50000;
             CurrentPage = currentPage;
+	        RevertIndex = revertIndex ?? false;
         }
 
         public int? CurrentPage { get; private set; }
 
+	    public bool RevertIndex { get; private set; }
```
Comment (uhaciogullari) [inline src/SimpleMvcSitemap.Sample/SampleBusiness/ProductSitemapConfiguration.cs]: Indentation

File: src/SimpleMvcSitemap.Sample/SampleBusiness/ProductSitemapConfiguration.cs [hunk 2]
```diff
@@ -7,15 +7,18 @@ public class ProductSitemapConfiguration : ISitemapConfiguration<Product>
     {
         private readonly UrlHelper _urlHelper;
 
-        public ProductSitemapConfiguration(UrlHelper urlHelper, int? currentPage)
+        public ProductSitemapConfiguration(UrlHelper urlHelper, int? currentPage, bool? revertIndex)
         {
             _urlHelper = urlHelper;
             Size = 50000;
             CurrentPage = currentPage;
+	        RevertIndex = revertIndex ?? false;
```
Comment (uhaciogullari) [inline src/SimpleMvcSitemap.Sample/SampleBusiness/ProductSitemapConfiguration.cs]: Indentation

File: src/SimpleMvcSitemap.Sample/SampleBusiness/SitemapConfiguration.cs [hunk 3]
```diff
@@ -6,15 +6,18 @@ public class SitemapConfiguration : ISitemapConfiguration<string>
     {
         private readonly UrlHelper _urlHelper;
 
-        public SitemapConfiguration(int? currentPage, UrlHelper urlHelper)
+        public SitemapConfiguration(int? currentPage, UrlHelper urlHelper, bool? revertIndex)
         {
             _urlHelper = urlHelper;
             CurrentPage = currentPage;
             Size = 1;
+	        RevertIndex = revertIndex ?? false;
```
Comment (uhaciogullari) [inline src/SimpleMvcSitemap.Sample/SampleBusiness/SitemapConfiguration.cs]: Indentation

File: src/SimpleMvcSitemap.Sample/SampleBusiness/SitemapConfiguration.cs [hunk 4]
```diff
@@ -6,15 +6,18 @@ public class SitemapConfiguration : ISitemapConfiguration<string>
...
         }
 
         public int? CurrentPage { get; private set; }
 
+	    public bool RevertIndex { get; private set; }
```
Comment (uhaciogullari) [inline src/SimpleMvcSitemap.Sample/SampleBusiness/SitemapConfiguration.cs]: Indentation

File: src/SimpleMvcSitemap.Tests/SitemapProviderTests.cs [hunk 5]
```diff
@@ -109,6 +109,7 @@ public void CreateSitemapWithConfiguration_NodeCountIsGreaterThanPageSize_Create
             FakeDataSource datas = new FakeDataSource().WithCount(5).WithEnumerationDisabled();
             _config.Setup(item => item.Size).Returns(2);
             _config.Setup(item => item.CurrentPage).Returns(currentPage);
+	        _config.Setup(item => item.RevertIndex).Returns(false);
```
Comment (uhaciogullari) [inline src/SimpleMvcSitemap.Tests/SitemapProviderTests.cs]: Indentation

File: src/SimpleMvcSitemap/SitemapProvider.cs [hunk 6]
```diff
@@ -107,12 +107,24 @@ private ActionResult CreateSitemapInternal(HttpContextBase httpContext, List<Sit
...
+			}
+			else
+			{
+				for (int page = pageCount; page <= pageCount; page--)
+				{
+					string url = configuration.CreateSitemapUrl(page);
+					SitemapIndexNode indexNode = new SitemapIndexNode {Url = url};
+					yield return indexNode;
+				}
+			}
```
Comment (uhaciogullari) [inline src/SimpleMvcSitemap/SitemapProvider.cs]: Indentation and duplicate code

**Comment by uhaciogullari** [comment]
You still have to fix the unit tests.

**Comment by fcuneyt** [comment]
Canceled previous pull request and send new one. Did you see last one?

**Comment by fcuneyt** [comment]
This is about my text editor settings. I only changed code side but auto formatted on commit i guess. I recommended you to develop this simple feature. It is only few rows of code. I don't know why you are choose this way.

**Comment by fcuneyt** [comment]
Ignore all of this. I resolve this locally

**Pull request closed without merging**